certified-connectors/SIGNL4 - Mobile Alerting/README.md

```diff
@@ -0,0 +1,34 @@
+# SIGNL4: Mobile Alerting and Incident Response
```

Comment:
@rons4 Please include all required sections in the readme file. please see out template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://[raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md)).

Reply:
Revised.
Thanks for your quick help and for your comments, @Amjed-Ayoub.